Prox/Prox/PlaceCarousel/PlaceCarouselHeaderView.swift

```diff
@@ -13,27 +13,30 @@ class PlaceCarouselHeaderView: HorizontalLineView {
         return label
     }()
 
+    fileprivate lazy var line: HorizontalLineView = HorizontalLineView()
+
     override init(frame: CGRect) {
         super.init(frame: frame)
         self.backgroundColor = .white
-        self.startX = 20.0
-        
-        self.setupViews()
-    }
 
-    // setting up the constraints for the views
-    func setupViews() {
-        color = Colors.carouselViewHeaderHorizontalLine
+        line.startX = 20.0
+        line.color = Colors.carouselViewHeaderHorizontalLine
+        line.backgroundColor = .clear
+        
         addSubview(numberOfPlacesLabel)
+        addSubview(line)
 
         let constraints = [ numberOfPlacesLabel.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 20),
-                            numberOfPlacesLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -10)]
+                            numberOfPlacesLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -10),
+                            line.bottomAnchor.constraint(equalTo: self.bottomAnchor),
+                            line.leadingAnchor.constraint(equalTo: self.leadingAnchor),
+                            line.trailingAnchor.constraint(equalTo: self.trailingAnchor),
+                            line.heightAnchor.constraint(equalToConstant: 1.5)]
```

Comment:
If we have to specify an explicit height anchor, it could make HorizontalLineView confusing to use in the future.

Add a comment to the class?